wallet/lib/pages/purchase_item/purchase_item_screen.dart

```diff
@@ -653,7 +659,7 @@ class _OwnerBottomDrawerState extends State<OwnerBottomDrawer> {
 
                                   final String? link = await generateLink(viewModel);
```

Comment:
Hide share button because if you dont have address how will you create a link with address.